src/index.css

```diff
@@ -1,27 +1,298 @@
+@import url(https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;600;700&family=Roboto+Slab:wght@300;400;500;700&display=swap);
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
+  padding: 4%;
+  background-color: #f5f6fa;
+}
+
+#root {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  padding: 0 0 4% 0;
+  border-radius: 4px;
+  height: auto;
+  width: 100%;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: #fafafa;
+}
+
+.top {
+  display: flex;
+  justify-content: space-between;
+  height: 7vh;
+  align-items: center;
+  width: 92%;
+  gap: 2vh;
+  background-color: #fff;
+  padding: 24px 4%;
+  border-bottom: solid 1px #e8e8e8;
+}
+
+.title {
+  display: flex;
+  gap: 2vw;
+  align-items: center;
+}
+
+.title h1 {
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #0290ff;
+}
```

Comment:
- [ ] Great effort working on the styling of the bookstore project 👍 👍, But currently your design requires some attention for it to be approved.

- Kindly add the profile icon to your design as seen in the original design, there is a profile icon on the right hand corner of the page, kindly add it.
- Also there is space between the input tags and the `add` button has a blue background color.

| Your Design | Original Design |
| ------------- | ------------- |
| ![book](https://github.com/JOdiemo/React-Bookstore-Project/assets/78634841/5e69115b-7071-4aab-a0e2-a75c25cdb369) | ![book1](https://github.com/JOdiemo/React-Bookstore-Project/assets/78634841/786ba7ef-7d30-4623-89d6-8d72d4cdd758) ![book2](https://github.com/JOdiemo/React-Bookstore-Project/assets/78634841/cc7d2b05-9f3a-4f51-b188-d426846df22a) |

- [ ] Optionally I would like you to work on the bookstore functionality, currently i notice when i add a book it duplicated the book and rendered it twice on the page. I marked it optional because it wasn't part of this milestone but i would greatly advice you work on it to have a good project.